### ISSUE TITLE
Pricing not updating using options and discounts

### ISSUE THREAD

**Issue by MajorPainage** [arastta/arastta#756]
If you have a product that has options, and some options increase the individual prices, and if your using discounts for bulk pricing, pricing doesn't update.  

eg.  So if you select the first option where it's the base price and no additional cost, let's say $1.00 and your discounts are 25+ is $0.75, and 50+ is $0.65.  Now if you select the 2nd option which increases the base price by $0.50.  The 2nd option makes the base price $1.50 which works fine, but if you select a higher quantity where the bulk pricing takes effect, the bulk pricing stays at $0.75 and $0.65, when it should also increase by $0.50 to 25+ at $1.25, and 50+ at $1.15.

So basically as it stands you can't use options where the price is adjusted and also use discount pricing for bulk discounts.
